pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -4,9 +4,13 @@
 
 import 'dart:async';
 import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
 
 import 'package:async/async.dart' hide Result;
 import 'package:collection/collection.dart';
+import 'package:coverage/coverage.dart';
+import 'package:path/path.dart';
```

Comment:
[nit] Almost all of our packages will import this with an alias: `p`

Reply:
I'll fix that when I fix tests!

Reply:
Fixed!